ui/src/components/app-sharing/app-sharing.tsx

```diff
@@ -187,16 +185,6 @@ export const AppSharing = ({
       <Stack direction="column">
         {currentUser?.share_permissions ? (
           <>
-            <Item>
-              <Alert
-                id="sharing-notification"
-                severity="warning"
-                icon={<WarningRoundedIcon sx={{ color: '#EAB54E' }} />}
-                sx={{ mb: '16px' }}
-              >
-                {message}
-              </Alert>
-            </Item>
```

Comment:
Why was this removed?

Reply:
It was one of the requirements in the definition of done.
<img width="452" alt="image" src="https://github.com/nebari-dev/jhub-apps/assets/885376/98304b05-a8e6-4137-aa0e-49cadada4df4">
https://gitlab.jatic.net/jatic/team-metrostar/t-e-platform/-/issues/643

Reply:
Added back in removed the word "Warning:"